.github/scripts/cherryPick.sh

```diff
@@ -0,0 +1,20 @@
+echo "Attempting to cherry-pick $1"
+
+CP_SUCCESS=false
```

Comment:
Do we need this variable? Why not exit 0 on the first if branch, else exit 0?

Reply:
Good point!